.github/workflows/test.yml

```diff
@@ -91,11 +92,14 @@ jobs:
           key: ${{ runner.os }}-go-docker-${{ matrix.go-version }}-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-docker-${{ matrix.go-version }}-
+      - name: Disable Long Tests
+        run: |
+          echo "OSMOSIS_E2E_SKIP_UPGRADE=true" >> $GITHUB_ENV
+        # This ensures that we disable long tests on all events other than push to main or release branches i.e. v9.x
+        if: ${{ !(github.event_name == 'push' && (github.ref == 'refs/heads/main' || (startsWith(github.ref,'refs/heads/v') && endsWith(github.ref, '.x')))) }}
       - name: Build Docker Image
```

Comment:
What do you think about, in addition to this, having some make commands accompany this? Maybe not for every combination, but maybe just have `make test-e2e-short` and `make test-e2e`, since keeping track of the various env variables maybe become more tedious as we add more (skipping upgrade, skipping state sync, skipping superfluid voting). Just a suggestion, open to other things